### PR TITLE
1 setup project framework

### PR DESCRIPTION
**Pull request by matt-auckland**

### CODE REVIEW

File: src/css/flexbox.css [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+}
+.grow-6 {
+  flex-grow: 6;        
+}
+
+.nowrap {
+  flex-wrap: nowrap;        
+}
+
+.width-100 {
```
Comment (matt-auckland) [inline src/css/flexbox.css]: unused classes here, will remove

**Comment by craigmacgregor** [comment]
I'm getting the select misalign with the text input. Rest looks good to me.

screen shot 2017-05-16 at 11 12 45 am

File: src/app/demo-sect/demo-sect.component.ts [hunk 2]
```diff
@@ -7,9 +7,10 @@ import { Component, OnInit } from '@angular/core';
 })
 export class DemoSectComponent implements OnInit {
 
+  coins = ['Navcoin', 'Bitcoin', 'Litecoin'];
```
Comment (craigmacgregor) [inline src/app/demo-sect/demo-sect.component.ts]: Should this be declared as const?

**Comment by craigmacgregor** [comment]
I'm getting the select misalignment with the text input. Chrome 57.0.2987.133 on OSX
screen shot 2017-05-16 at 11 12 45 am

**Pull request merged**